### ISSUE TITLE
webp文件系统iOS12下载加载图片失败

### ISSUE THREAD

**Issue by dreamahui** [rs/SDWebImage#2498]
### New Issue Checklist
iOS12.0系统

使用webp格式图片资源，在本地加载失败显示不出图片

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 11.0 / 10.13.0 / 11.0 / 4.0
 SDWebImage Version      | e.g. 4.2.0 / 4.1.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 9 / Xcode 8
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by dreamahui** [comment]
请问怎么解决

**Comment by dreampiggy** [comment]
Duplicated for #2347 . It has been solved in the #2348 and be available in [4.4.1](https://github.com/rs/SDWebImage/releases/tag/4.4.1). Please upgrade your podfile to use the new version.

```ruby
pod 'SDWebImage', '~> 4.4.1'
```
